src/components/widgets/ConfirmationDialog.tsx

```diff
@@ -41,15 +41,41 @@ const useStyles = makeStyles(theme => ({
     lineHeight: '19px',
   },
   confirmButton: {
-    backgroundColor: theme.palette.error.main,
+    width: '74px',
+    height: '49px',
+    background: '#FCD2D2',
+    boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)',
+    borderRadius: '0px',
+    '&:hover': {
+      transform: 'translateY(1px)',
+      backgroundColor: '#EDC6C6',
+    },
+  },
+  dialogPaper: {
+    width: '302px',
+    height: '275px',
+  },
+  cancelButton: {
+    width: '120px',
+    height: '49px',
+    outline: 'none',
+    borderRadius: '0px',
+    borderColor: 'black',
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    background: '#FCFCFC',
+    boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)',
+    '&:hover': {
+      transform: 'translateY(1px)',
+    },
   },
 }))
 
 type ConfirmationDialogProps = {
   isOpen: boolean
   type: 'DELETE' | 'NAVIGATE'
   title?: string
-  cancelText?: string
+  createNewStudyText?: string
```

Comment:
This confirmation dialog is used throughout the app - not just for deleting studies.  If you want to change the value of the props you need to do this in the parent component. If you need props that are not there -- we need to add additional for make it more flexible